lib/testrbl.rb

```diff
@@ -2,9 +2,9 @@
 
 module Testrbl
   PATTERNS = [
-    /^(\s+)(should|test|it)\s+['"](.*)['"]\s+do\s*(?:#.*)?$/,
-    /^(\s+)(context|describe)\s+['"]?(.*?)['"]?\s+do\s*(?:#.*)?$/,
-    /^(\s+)def\s+(test_)([a-z_\d]+)\s*(?:#.*)?$/
+    /^(\s+)(should|test|it)(\s+|\s*\(\s*)['"](.*)['"](\s*\))?\s+do\s*(?:#.*)?$/,
```

Comment:
I was thinking:
```suggestion
    /^(\s+)(should|test|it)\s+['"(](.*)['")]\s+do\s*(?:#.*)?$/,
```
since  nobody does `it ( "foo" )` anyway 🤷

Reply:
The tests I found this on do `it('foo') do`. Some even do `it('foo') { code }`, which this doesn't yet address.

Some key differences with your suggestion are

1. The paren isn't an alternative for the quotes, so a multiplier would have to be added
1. We would need to change the whitespace to allow zero whitespace